src/app/routes/SettingsPage.test.tsx

```
@@ -175,166 +182,61 @@ describe("SettingsPage", () => {
-      await screen.findByText(/experimental_feature_flag/),
-    ).toBeInTheDocument();
-
-    fireEvent.click(screen.getByRole("button", { name: "Validate TOML" }));
+    expect(await screen.findByText("Used: 41%")).toBeInTheDocument();
 
-    await waitFor(() => {
-      expect(validateConfigMock).toHaveBeenCalledWith("/repo/theoden");
-    });
-    const validationErrors = await screen.findAllByText(
-      "Unknown key experimental_feature_flag",
+    mocks.notificationListeners.forEach((listener) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)